src/main/java/org/apache/accumulo/access/Authorizations.java

```diff
@@ -58,9 +59,12 @@ public int hashCode() {
     return authorizations.hashCode();
   }
 
+  /**
+   * @return a String containing the sorted, comma-separated set of authorizations
+   */
   @Override
   public String toString() {
-    return authorizations.toString();
+    return authorizations.stream().sorted().collect(Collectors.joining(","));
```

Comment:
The set to toString will put `[` and `]` around data which is nice.   Could possibly add those.

Reply:
I have no issue with this, but I'm curious on the difference between this and `new TreeSet<>(authorizations).toString()`

Reply:
Using a TreeSet will put a space after each comma and add the brackets. e.g. `[A, B, Y, Z]`

Using the stream with `.joining(",")` will not add the spaces nor the brackets by default. e.g. `A,B,Y,Z` but the spaces and brackets can be added obviously.

The TreeSet code would be more simple I think but also would have the overhead of creating the new TreeSet (which might be negligible in this case).

Reply:
Creating a treeset seems fine for this case.  Thinking is that when toString is called its for the purpose of showing a person something, so in that case the expense is worthwhile.  Can optimize later if ever needed.

Reply:
> The TreeSet code would be more simple I think but also would have the overhead of creating the new TreeSet

I think that assumes that the streaming example creates no objects. 

`Collection.stream` calls `StreamSupport.stream`, which creates a new `ReferencePipeline.Head`:
https://github.com/openjdk/jdk/blob/jdk-11%2B28/src/java.base/share/classes/java/util/stream/StreamSupport.java#L67

Calling `sorted` on the `ReferencePipeline` creates more objects:
https://github.com/openjdk/jdk/blob/jdk-11%2B28/src/java.base/share/classes/java/util/stream/ReferencePipeline.java#L462

`Collectors.joining` creates two new objects.

https://github.com/openjdk/jdk/blob/jdk-11%2B28/src/java.base/share/classes/java/util/stream/Collectors.java#L367

Reply:
Made toString use TreeSet in 51e02f5